doc/release/README.chplenv.rst

```diff
@@ -111,6 +111,8 @@ MANPATH
 Optional Settings
 -----------------
 
+.. _chpl_target_platform:
```

Comment:
Eh, I think it might be reasonable for other files to link to them, so I think I'll leave them as is